pkg/operator/resources/validations.go

```diff
@@ -87,26 +88,42 @@ func ValidateClusterAPIs(apis []userconfig.API, projectFiles spec.ProjectFiles)
 
 	didPrintWarning := false
 
+	withoutAPISplitter := InclusiveFilterAPIsByKind(apis, userconfig.SyncAPIKind)
 	for i := range apis {
 		api := &apis[i]
-		if err := spec.ValidateAPI(api, projectFiles, types.AWSProviderType, config.AWS); err != nil {
-			return err
-		}
-		if err := validateK8s(api, virtualServices, maxMem); err != nil {
-			return err
-		}
+		if api.Kind == userconfig.SyncAPIKind {
+			if err := spec.ValidateAPI(api, projectFiles, types.AWSProviderType, config.AWS); err != nil {
+				return errors.Wrap(err, api.Identify())
+			}
+			if err := validateK8s(api, virtualServices, maxMem); err != nil {
+				return errors.Wrap(err, api.Identify())
+			}
 
-		if !didPrintWarning && api.Networking.LocalPort != nil {
-			fmt.Println(fmt.Sprintf("warning: %s will be ignored because it is not supported in an environment using aws provider\n", userconfig.LocalPortKey))
-			didPrintWarning = true
+			if !didPrintWarning && api.Networking.LocalPort != nil {
+				fmt.Println(fmt.Sprintf("warning: %s will be ignored because it is not supported in an environment using aws provider\n", userconfig.LocalPortKey))
+				didPrintWarning = true
+			}
+			dups := spec.FindDuplicateNames(withoutAPISplitter)
```

Comment:
This duplicate name check isn't required because it should be handled by the all apis duplicate check on line 123.